libs/blocks/global-navigation/global-navigation.js

```diff
@@ -101,27 +98,28 @@ class Gnav {
     // eslint-disable-next-line no-async-promise-executor
     this.ready = this.ready || new Promise(async (resolve) => {
       this.el.removeEventListener('click', this.loadDelayed);
+      // TODO load placeholders here? is that too late?
```

Comment:
I don't think we should force-load placeholders, each component could request them as needed.

Reply:
Ah yes, this is a very early relict before I looked deeper into that - removed 👍